spec/stateful/signals.py

```diff
@@ -105,6 +105,7 @@ def sigmultiple_trapped_prompt(sh):
   expect_prompt(sh)
 
   sh.kill(signal.SIGUSR1)
+  time.sleep(0.1)
```

Comment:
Add a comment here about this

I guess the shell does the right thing, but it's hard to test the non-deterministic output?

Reply:
Yeah I think so. You always see the output of both traps, just sometimes out of order